recipes/mwclient/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: mwclient
+  version: "0.8.4"
+
+source:
+  fn: mwclient-0.8.4.tar.gz
+  url: https://pypi.python.org/packages/66/84/e99c39f3ab25a18eea9371d4e47ba79866d08f2e0ef3d70926f3ebd05c1b/mwclient-0.8.4.tar.gz
+  md5: 8532c9e0e0cbda4e1b6495c361d0310c
+
+build:
+  number: 0
```

Comment:
Please add the build script
```yaml
script: python setup.py install --single-version-externally-managed --record=record.txt
```

I am :confused: as to why this recipe was not failing on all of the CI services? Any ideas @ocefpaf? Since there was no setup.py invocation, how was the test/imports section importing mwclient?